app/javascript/react/components/molecules/Calendar/HeaderToggle/HeaderToggle.tsx

```diff
@@ -42,42 +46,66 @@ const HeaderToggle = ({
     return (
       <S.ResetButton onClick={resetThresholds}>
         {t("thresholdConfigurator.resetButton")}
-        <img src={returnArrow} alt={t("thresholdConfigurator.altResetButton")} />
+        <img
+          src={returnArrow}
+          alt={t("thresholdConfigurator.altResetButton")}
+        />
       </S.ResetButton>
     );
   };
 
+  const displayMapPageResetButton = () => {
+    if (!resetThresholds) return null;
+    return (
+      <S.ThresholdResetButton onClick={resetThresholds}>
+        <img
+          src={returnArrow}
+          alt={t("thresholdConfigurator.altResetButton")}
+        />
+      </S.ThresholdResetButton>
+    );
+  };
+
   return (
     <>
-      <S.Container>
-        {(isMobile && (
-          <>
-            <S.RotatedIcon
-              src={headerArrowIcon}
-              alt={t("headerToggle.arrowIcon")}
-              rotated={!isVisible}
-              onClick={toggleVisibility}
-            />
-            <S.Heading onClick={toggleVisibility}>{titleText}</S.Heading>
-          </>
-        )) || 
-        <S.Heading>
-          {titleText}
-          {startDate && endDate && (
+      {isMobile && !isMapPage && (
```

Comment:
We can add some information to "treshold ticket" to remember to simplify this logic 💪